### ISSUE TITLE
Improve settings page

### ISSUE THREAD

**Issue by lucca65** [cambiatus/frontend#751]
Our current settings page look like this: 

<img width="1310" alt="Captura de Tela 2022-05-20 às 14 59 42" src="https://user-images.githubusercontent.com/1082127/169586470-53d3b562-1856-4468-9b9b-b15436e323c8.png">

It makes a little bit hard to navigate as all functionality is scattered around. We could review the hierarchy and improve how settings are accessible on the community, to make the flow easier.

Some examples: 

<img width="868" alt="Captura de Tela 2022-05-20 às 15 02 29" src="https://user-images.githubusercontent.com/1082127/169586895-de96d2e7-1e18-4646-a03c-8388246abb14.png">

<img width="1155" alt="Captura de Tela 2022-05-20 às 15 03 05" src="https://user-images.githubusercontent.com/1082127/169586965-2076361d-11ff-4775-aab4-e361174a6a9c.png">

<img width="717" alt="Captura de Tela 2022-05-20 às 15 03 39" src="https://user-images.githubusercontent.com/1082127/169587037-e9ea4881-7293-4460-b14b-d78535a29f5c.png">

<img width="782" alt="Captura de Tela 2022-05-20 às 15 04 12" src="https://user-images.githubusercontent.com/1082127/169587130-fdb7d826-b837-4634-a2e7-750f9ddff074.png">

**Comment by lucca65** [comment]
@juramos-2020 @rafachadud podem nos ajudar com isso? Não é prioridade, mas seria legal começarmos a ver algo nesse sentido agora que vamos ter mais telas de configuração com a entrada de funcionalidades do Shop e depois de roles. Já tá difícil de usar hoje sem essas funcionalidades

**Comment by juramos-2020** [comment]
OI @lucca65 

essa issue seria uma melhoria [dessa issue](https://github.com/cambiatus/frontend/issues/726)?

**Comment by juramos-2020** [comment]
Caso seja uma melhoria, adorei as referencias isso vai ajudar muito! 🙏

Para ajudar na formulação de uma proposta, gostaria de pedir uma ajudinha. Você pode responder as perguntas abaixo? 
Eu comecei respondendo, mas gostaria do seu complemento 😉👍

**Essa melhoria vai resolver qual problema?**
- diminuir os espaços vazio
- dar mais opções de configurações

**Minha expectativa com ela é de...**
- deixar as opções de configurações mais visíveis

**Comment by lucca65** [comment]
Sim, é na mesma tela, mas acho que vai mais longe né? Por mim poderíamos fazer uma melhoria só já com os painéis repensados em uma estrutura melhor.... Oq acham @juramos-2020 @NeoVier

**Comment by lucca65** [comment]
**Essa melhoria vai resolver qual problema?**
- As configurações são muito confusas de acessar hoje
- Muitas configurações ficam sem hierarquia. Comunicados está no mesmo nível de Informações da comunidade. 


**Minha expectativa com ela é de...**
- deixar as opções de configurações mais visíveis
- Tornar a visualização das comunidades mais fácil, parecida e familiar com outros apps com configurações tão complexas quanto a Cambiatus
- Tornar claro aos administradores onde achar cada coisa. Onde mudo informações do Token? Onde faço moderação do shop? Onde mudo informações da comunidade.

Por exemplo, informações da comunidade é um termo bem amplo. Posso entender nisso, contatos, nome, foto. Mas na tela atual, é ali que mudamos o valor mínimo que podemos ficar negativos e recompensa de convite (informação do token)

Habilitar funcionalidades, como a outra issue que você mencionou trabalha, também fica oculta. No geral temos um monte de cards esticados, fica difícil entender como eles se relacionam, principalmente se você comparar com outros exemplos e referencias que mandei.

**Comment by lucca65** [comment]
- [ ] a
- b
- c
